staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -112,7 +119,10 @@ type VirtualMachineInstanceSpec struct {
 	//
 	// +optional
 	EvictionStrategy *EvictionStrategy `json:"evictionStrategy,omitempty"`
-
+	// StartStrategy can be set to "Paused" if Virtual Machine should be started in paused state.
+	//
+	// +optional
+	StartStrategy *StartStrategy `json:"startStrategy,omitempty"`
```

Comment:
@stu-gott what do you think about this? 

This new field would be specific to VMIs and it impacts how the VMI is started.  It sounds kind of similar to RunStrategy, which is just for VMs though.